ietf/static/css/ietf.scss

```diff
@@ -909,13 +909,7 @@ td.position-empty {
     padding-left: 0px !important;
 }
 
-.ongoing>td:first-child {
-    background-color: red !important;
-}
-
-.ongoing>td:last-child {
-    background-color: red !important;
-}
+#agenda-table tbody tr.current-session { border-top: 0.2rem solid map.get($theme-colors, "info"); }
```

Comment:
Is possible, use bs5 variables for `border-top`?

Reply:
Good idea, I'll give that a try.

Reply:
This appears to work well - `map($border-widths, 2)` is a close match to what I eyeballed.